pkgs/development/python-modules/scikit-learn-extra/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, numpy
+, cython
+, scipy
+, scikit-learn
+, matplotlib
+, pytestCheckHook
+}:
```

Comment:
nit: alphabetize

Reply:
Not really sure if alphabetic order is better than somewhat semantic grouping (libraries/library functions → *Inputs → options).